OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -93,15 +93,16 @@ public class Armament : UpgradableTrait<ArmamentInfo>, ITick, IExplodeModifier
 		public readonly Barrel[] Barrels;
 
 		readonly Actor self;
-		Lazy<Turreted> turret;
+		Turreted turret;
+		AmmoPool ammoPool;
 		Lazy<BodyOrientation> coords;
-		Lazy<AmmoPool> ammoPool;
 		List<Pair<int, Action>> delayedActions = new List<Pair<int, Action>>();
 		Lazy<IEnumerable<int>> rangeModifiers;
```

Comment:
I wasn't sure whether that could cause any regressions. If you say it won't, I'll unlazy them, too.